pyproject.toml

```diff
@@ -12,8 +12,11 @@ dependencies = [
     "pose-format",
     "tqdm",
     "mediapipe",
+    "joeynmt @ git+https://github.com/may-/joeys2t.git@4532e8b9b990c8a3c4ccfe9fbd141bddc6d8637b",
```

Comment:
this should go under `tool.poetry.extras`, not in the main dependencies.
This helps understanding what dependencies are needed for each implementation